velox/functions/prestosql/registration/ComparisonFunctionsRegistration.cpp

```diff
@@ -43,6 +43,18 @@ void registerComparisonFunctions() {
   registerFunction<BetweenFunction, bool, Varchar, Varchar, Varchar>(
       {"between"});
   registerFunction<BetweenFunction, bool, Date, Date, Date>({"between"});
+  registerFunction<
```

Comment:
Will the existing framework ensure the arguments have the same precision and scale?